### PR TITLE
Use async test helper to simplify node testing

### PR DESCRIPTION
**Pull request by cecton**

### CODE REVIEW

File: node/test-service/tests/build-blocks.rs [hunk 1]
```diff
@@ -41,38 +33,21 @@ async fn ensure_test_service_build_blocks() {
...
-	pin_mut!(t1, t2);
-
-	select! {
-		_ = t1 => {
-			panic!("the test took too long, maybe no blocks have been produced");
-		},
-		_ = t2 => {},
-	}
+	alice.task_manager.clean_shutdown();
+	bob.task_manager.clean_shutdown();
```
Comment (cecton) [inline node/test-service/tests/build-blocks.rs:52]:
```suggestion
	alice.task_manager.clean_shutdown().await;
	bob.task_manager.clean_shutdown().await;
```

File: node/test-service/tests/call-function.rs [hunk 2]
```diff
@@ -14,60 +14,31 @@
...
-
-	pin_mut!(t1, t2);
 
-	select! {
-		_ = t1 => {
-			panic!("the test took too long, maybe no blocks have been produced");
-		},
-		_ = t2 => {},
-	}
+	alice.task_manager.clean_shutdown();
```
Comment (cecton) [inline node/test-service/tests/call-function.rs:43]:
```suggestion
	alice.task_manager.clean_shutdown().await;
```

**Comment by seunlanlege** [comment]
I guess you don't need to add the env variable to the ci scripts?

**Comment by cecton** [comment]
> I guess you don't need to add the env variable to the ci scripts?

Nope! It's using the default value on 600

**Comment by cecton** [comment]
bot merge

**Comment by cecton** [comment]
bot merge PLEASE

**Comment by cecton** [comment]
bot merge

**Comment by cecton** [comment]
bot merge

**Pull request merged**